docs/api/metrics/deletion.md

```diff
@@ -1,7 +1,6 @@
 # Deletion
 
-The Deletion Fidelity metric measures how well a saliency-map–based explanation of an image classification result localizes
-the important pixels.
+The Deletion Fidelity metric measures how well a saliency-map–based explanation localizes the important features.
```

Comment:
This description would be true for Insertion too. I think we could specify "by iteratively deleting the most important features while tracking the prediction score" or something like that!

Reply:
Okay considering what follows maybe just saying "a score" would do